### PR TITLE
feature/138-release-history

### PR DESCRIPTION
**Pull request by jamesmelzer**
## 1.0.0
    [View Docs](/v/1.0.0)
    ##### Released 8/27/2018
    #### Visual Style 
    * [Color](/visual-style/color.html) - Guidelines and tokens published
    * [Icons](/visual-style/icons.html) - Guidelines, tokens, and assets published
    * [Space](/visual-style/space.html) - Guidelines and tokens published
    * Elevation - Tokens defined (documentation pending)
    * Typography - Tokens defined (documentation pending)
    #### Updated Components 
    * [Buttons](/components/buttons.html) - Guidelines published
    #### New Components
    * Cards - Code and Sketch only (documentation pending)
    * Containers - Code and Sketch only (documentation pending)
    * Forms - Code and Sketch only (documentation pending)
    * Feedback Messages - Code and Sketch only (documentation pending)
    * Grids - Code and Sketch only (documentation pending)
    * Links - Code and Sketch only (documentation pending)
    * Lists - Code and Sketch only (documentation pending)
    * Long Form Text - Code and Sketch only (documentation pending)
    * Tiles - Code and Sketch only (documentation pending)
    * Vertical Navigation - Code and Sketch only (documentation pending)
    #### Assets
    * [Release History](/release-history.html) - Published

### CODE REVIEW

**Comment by jamesmelzer** [comment]
@joesnellpdx 
1. It's not important that we keep the GDoc release history. Since we're stacking up a lot of releases in a short period, I'm being intentionally specific and including 1.0, 1.1, etc. 
2. Correct. I've drafted it for you in a PR. You can certainly make edits if the scope changes before you release. 
3. Hmm... you're correct. That's just the sort of update you could make to my PR :)
4. Agreed in all releases except this one, since it culminates two months work in a single release (as far as the history is concerned.

File: doc-site/docs/release-history.njk [hunk 1]
```diff
@@ -6,38 +6,66 @@
 {% endblock %}
 
 {% block page_shell_content %}
-  {% filter markdown %}    
+  {% filter markdown %}
+    ## 1.0.0-alpha-1
+    [View Docs](/v/1.0.0-alpha-1)
```
Comment (joesnellpdx) [inline doc-site/docs/release-history.njk]: @jamesmelzer 

Found one more issue. The 'view docs' link does not go anywhere. Is that for a later release or does that need to be fixed?

Comment (kevinmpowell) [reply doc-site/docs/release-history.njk]: @joesnellpdx it _will_ go somewhere once you walk through the release process. `/v/1.0.0-alpha-1` is created as part of the `gulp deploy-to-gh-pages` task

**Comment by kevinmpowell** [comment]
> are we keeping release history documentation in gdrive at all ( see following link - I assume that is just for the initial doc site release and not for records), or will it all be in code? I think all in code.
https://docs.google.com/document/d/1k_dkP0kfhNpg-r5rUnxuYEfG2fEErLePwyLi3isVz5s/edit#heading=h.8nxnd33071na

I'm on the fence about this. Occasionally Release Histories contain content that needs to be editorially edited and that is far better suited to Google Drive than a PR in Github.

File: doc-site/docs/release-history.njk (hunk 1 above)
Comment (jamesmelzer) [reply doc-site/docs/release-history.njk]: Just gotta make sure the nomenclature is identical throughout the process. This release will be fussier than most, which are just x.x.x

**Comment by kevinmpowell** [comment]
I'm less on the fence the more I think about it. This PR _should_ appear as part of @joesnellpdx's release process PR for doc-site after he _migrated_ content from Google Drive into the repo. 

My expectation for this task is that @jamesmelzer would be requesting review of his Google Doc not a PR review.

I recommend release history be drafted/collected in google drive and that the person making code changes for the release actually perform the task of moving it over.

**Comment by joesnellpdx** [comment]
@kevinmpowell @jamesmelzer 

In that case, we should set up a directory for releases in G-drive and add to the documentation with link to directory.

**Comment by kevinmpowell** [comment]
To that end I've left feedback in G-Drive and I have some open questions to consider before moving forward with this release history.

**Pull request merged**